badge/src/main/java/com/decathlon/vitamin/compose/badge/items/VitaminCountBadge.kt

```diff
@@ -0,0 +1,103 @@
+package com.decathlon.vitamin.compose.badge.items
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import com.decathlon.vitamin.compose.badge.models.BadgeSize
+import com.decathlon.vitamin.compose.badge.models.BadgeStyle
+
+@Composable
+internal fun VitaminCountBadge(
+    count: Int,
+    badgeStyle: BadgeStyle,
+    nbMaxChar: Int,
+    badgeSize: BadgeSize,
+    content: @Composable BoxScope.() -> Unit
+) {
+    if (count == 0) {
+        Box(content = content)
+    } else {
+        VitaminBadge(
+            content = content,
+            count = count,
+            badgeStyle = badgeStyle,
+            nbMaxChar = nbMaxChar,
+            badgeSize = badgeSize
+        )
+    }
+}
+
+@Composable
+internal fun VitaminCountBadge(
+    modifier: Modifier = Modifier,
+    count: Int,
+    badgeStyle: BadgeStyle,
+    nbMaxChar: Int,
+    isHiddenCountLeft: Boolean,
+    badgeSize: BadgeSize
+) {
+    if (count > 0) {
+        val radius = badgeSize.radius
+        val shape = RoundedCornerShape(radius)
+        Row(
+            modifier = modifier
+                .defaultMinSize(minWidth = radius * 2, minHeight = radius * 2)
+                .background(
+                    color = badgeStyle.backgroundColor,
+                    shape = shape
+                )
+                .clip(shape)
+                .padding(
+                    horizontal = badgeSize.horizontalPadding, vertical = badgeSize.verticalPadding
+                ),
+            verticalAlignment = Alignment.CenterVertically,
+            horizontalArrangement = Arrangement.Center
+        ) {
+            Text(
+                text = if (isHiddenCountLeft) "+$count" else getFormatedCount(
```

Comment:
I don't see this behavior on decathlon.design or in the Android UI KIt on Figma. :thinking: